docs/docs/userguide/configuration/CONFLUENCE_CONFIGURATION.md

```diff
@@ -109,4 +109,48 @@ You may want to adjust these values according to your needs. For more informatio
 
 ```terraform
 confluence_collaborative_editing_enabled = true
+```
+
+## Dataset restore configuration
+To restore the dataset into the newly created instance, configure the following parameters.
+
+### Database Snapshot Identifier
+
+`confluence_db_snapshot_identifier` sets the identifier of the DB snapshot to restore from. If you do not specify a value, no AWS RDS snapshot will be used.
+
+```terraform
+confluence_db_snapshot_identifier = "<SNAPSHOT_IDENTIFIER>"   # e.g. "my-snapshot"
+```
+
+!!! info "The AWS RDS snapshot must be in the same region and account as the RDS instance to be created."
+
+    You also need to provide the master user credentials (`confluence_db_master_username` and `confluence_db_master_password`) that match the snapshot.
+
+!!! tip "Optimise the restore performance."
+
+    To obtain the best performance, configure Jira RDS that match the snapshot including `confluence_db_instance_class` and `confluence_db_allocated_storage`.
+
+### Database Master Username
+
+`confluence_db_master_username` sets the username for the RDS master user. If you do not specify a value, username is "postgres".
+
+```terraform
+confluence_db_master_username = "<DB_MASTER_USERNAME>"   # e.g. "postgres"
+```
+
+### Database Master Password
+
+`confluence_db_master_password` sets the password for the RDS master user. If you do not specify a value, a random password will be generated.
+
+```terraform
+confluence_db_master_password = "<DB_MASTER_PASSWORD>"   # default value is null
+```
+
+### Build Number
+
+`confluence_db_snapshot_build_number` configures Confluence instance with the correct build number that stores in the snapshot.
```

Comment:
Maybe put the link of list of build numbers here so users can look up their products' build number

Reply:
It is worth to mention this variable is mandatory if snapshot dataset is provided and otherwise will be ignored.

Reply:
@nghazali Modified line 115 to indicate all the parameters in 'Database restore configuration' section are mandatory.